pkg/execution/state/redis_state/queue.go

```diff
@@ -2062,19 +2158,15 @@ func (q *queue) Scavenge(ctx context.Context) (int, error) {
 	// Each of the items is a concurrency queue with lost items.
 	var resultErr error
 	for _, partition := range pKeys {
-		// Fetch the partition.  This uses the concurrency:p: prefix,
-		// so remove the prefix from the item.
-		partitionJSON, err := q.u.unshardedRc.Do(ctx, q.u.unshardedRc.B().Hget().Key(q.u.kg.PartitionItem()).Field(partition).Build()).AsBytes()
-		if err == rueidis.Nil {
-			continue
-		}
-		if err != nil {
-			resultErr = multierror.Append(resultErr, fmt.Errorf("error finding partition '%s' during scavenge: %w", partition, err))
-			continue
+
+		// If this is a UUID, assume that this is an old partition queue
+		queueKey := partition
+		if len(partition) == 36 {
```

Comment:
(minor) I've seen this magic number a couple times now and it would be nice to either:

1. define an `isStringUUID()` function for internal use within this package
2. extract `36` to a constant